lib/src/views/misc/interactive_grid_tile.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:rescado/src/views/buttons/floating_button.dart';
+
+class InteractiveGridTile extends StatefulWidget {
+  final String image;
+  final FloatingButton floatingButton;
+
+  const InteractiveGridTile({Key? key, required this.image, required this.floatingButton}) : super(key: key);
+
+  @override
+  _GridTileState createState() => _GridTileState();
+}
+
+class _GridTileState extends State<InteractiveGridTile> {
+  bool showAction = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+        onLongPress: () => setState(() => showAction = !showAction),
+        child: Stack(
+          children: [
+            AnimatedOpacity(
```

Comment:
So... does this animation work? 😛  Perhaps scale+opacity is better than just opacity. Just a thought.

Reply:
Dit werkt ja. Je kan geen animatie op een DecorationImage doen.
Daarom werkte dat vorige keer niet.